charts/m4d-crd/templates/app.m4d.ibm.com_m4dapplications.yaml

```diff
@@ -218,7 +218,24 @@ spec:
                     description: Reference to a secret where the credentials are stored
                     type: string
                 type: object
-              description: ProvisionedStorage maps a dataset (identified by DataSetID) to the new provisioned bucket. It allows M4DApplication controller to manage buckets in case the spec has been modified, an error has occurred, or a delete event has been received. ProvisionedStorage has the information required to register the dataset once the owned plotter resource is ready
+              description: ProvisionedStorage maps a dataset (identified by AssetID) to the new provisioned bucket. It allows M4DApplication controller to manage buckets in case the spec has been modified, an error has occurred, or a delete event has been received. ProvisionedStorage has the information required to register the dataset once the owned plotter resource is ready
+              type: object
+            readEndpointsMap:
+              additionalProperties:
+                description: The spec of the endpoint exposed by the module
+                properties:
+                  Scheme:
```

Comment:
is this schema or protocol?

Reply:
URI scheme, see https://en.wikipedia.org/wiki/List_of_URI_schemes